portal-api/src/v1/gef/services/clone/helpers/generate-cloned-deal.test.ts

```diff
@@ -0,0 +1,146 @@
+import { aPortalSessionUser, Bank, Deal, DEAL_STATUS, getCurrentGefDealVersion } from '@ukef/dtfs2-common';
+import { ObjectId } from 'mongodb';
+import { generateAuditDatabaseRecordFromAuditDetails, generateSystemAuditDetails } from '@ukef/dtfs2-common/change-stream';
+import { generateClonedExporter } from './generate-cloned-exporter';
+import { generateClonedDeal, propertiesToRemove } from './generate-cloned-deal';
+
+const mockLatestEligibilityCriteria = {
+  eligibility: 'criteria',
+};
+
+const existingDealId = new ObjectId().toString();
+const mockAuditDetails = generateSystemAuditDetails();
+const mockBank = {
+  id: 'bank id',
+} as Bank;
+const bankInternalRefName = 'bankInternalRefName';
+const additionalRefName = 'additionalRefName';
+const maker = aPortalSessionUser();
+
+const existingDeal: Deal = {
```

Comment:
could this come from test-mocks? (please ignore if this has been moved across)

Reply:
I replied to Tony's comment here about this https://github.com/UK-Export-Finance/dtfs2/pull/3782/files/bb651262b4edbd25b55bb131b309bbfcef2c354f#r1824772073 

The TLDR is that I cant see any good mock deal fixtures & cloning a deal changes a lot of these values so I want to be explicit in these tests